src/main/java/com/github/lunatrius/laserlevel/json/MarkerSerializer.java

```diff
@@ -42,6 +43,7 @@ public Marker deserialize(final JsonElement json, final Type typeOfT, final Json
         if (json.isJsonObject()) {
             try {
                 final JsonObject object = json.getAsJsonObject();
+                final int maxdistance = object.getAsJsonPrimitive("maxdistance").getAsInt();
```

Comment:
```java
final int maxDistance = object.has("maxdistance") ? object.getAsJsonPrimitive("maxdistance").getAsInt() : 64;
```
Otherwise it will "delete" all old entries.